hack/unit-test.sh

```diff
@@ -9,13 +9,15 @@ unit_test() {
     cd ../ || return
     go test -i ./...
 
-    cmd="go list ./... | grep 'github.com/dragonflyoss/Dragonfly/'"
+    # folder /test contains test cases for integration test.
+    # then we exclude them in unit test.
+    filterCmd="go list ./... | grep -v 'github.com/dragonflyoss/Dragonfly/test'"
     sources="${GO_SOURCE_DIRECTORIES[*]}"
     sources="${sources// /|}"
     retCode=0
-    test -n "${sources}" && cmd+=" | grep -E '${sources}'"
+    test -n "${sources}"
```

Comment:
as @fengzixu said, we can change `GO_SOURCE_DIRECTORIES` to `GO_SOURCE_EXCLUDES`, and use `grep -v` to exclude what we don't want to test:
```bash
    cmd="go list ./... | grep 'github.com/dragonflyoss/Dragonfly/'"
    excludes="${GO_SOURCE_EXCLUDES[*]}"
    excludes="${excludes// /|}"
    retCode=0
    test -n "${excludes}" && cmd+=" | grep -vE '${excludes}'"
```

Reply:
updated, quite reasonable